packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol

```diff
@@ -138,7 +153,28 @@ contract L1StandardBridge_Pause_TestFail is Bridge_Initializer {
     }
 }
 
-contract L1StandardBridge_Initialize_TestFail is Bridge_Initializer { }
+contract L1StandardBridge_Initialize_TestFail is Bridge_Initializer {
+    /// @dev Tests that the proxy cannot be initialized twice.
+    function test_initializeProxy_alreadyInitialized_reverts() external {
```

Comment:
if we update `Initializable.sol` test then we can delete this test case since it is already covered there